benchmarks/pom.xml

```diff
@@ -220,6 +220,13 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <configuration>
+          <skip>true</skip> <!-- ignore non-production code -->
```

Comment:
Should the integration-tests be skipped too?

Reply:
Jacoco coverage isn't collected for the integration tests.